client/src/components/SelectButton/index.tsx

```diff
@@ -0,0 +1,35 @@
+interface SelectButtonProps {
+  children: React.ReactNode;
+  size?: 'default' | 'md' | 'lg';
+  type: 'default' | 'active' | 'IconActive';
+  onClick?: () => void;
+}
+
+const selectSize = {
+  default: 'py-6px',
+  md: 'w-166px h-60px py-20px',
+  lg: 'w-343px h-60px py-20px',
+  full: 'w-full h-60px py-20px',
+};
```

Comment:
```suggestion

type SelectSize = 'default' | 'md' | 'lg' | 'full';

const selectSize: Record<SelectSize, string> = {
  default: 'py-6px',
  md: 'w-166px h-60px py-20px',
  lg: 'w-343px h-60px py-20px',
  full: 'w-full h-60px py-20px',
};
```

요렇게 야무지게 타이핑할 수 있습니닷

Reply:
오 역시 TS 권위자.. 꿀팁 감사요!!